controllers/doguDeleteManager_test.go

```diff
@@ -60,13 +63,19 @@ func TestNewDoguDeleteManager(t *testing.T) {
 	t.Run("success", func(t *testing.T) {
 		// given
 		// override default controller method to retrieve a kube config
-		oldGetConfigOrDieDelegate := ctrl.GetConfigOrDie
-		defer func() { ctrl.GetConfigOrDie = oldGetConfigOrDieDelegate }()
-		ctrl.GetConfigOrDie = func() *rest.Config {
-			return &rest.Config{}
+		oldGetConfigDelegate := ctrl.GetConfig
+		defer func() { ctrl.GetConfig = oldGetConfigDelegate }()
+		ctrl.GetConfig = createTestRestConfig
+
+		addImagesConfigmap := &v1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      util.OperatorAdditionalImagesConfigmapName,
+				Namespace: testNamespace,
+			},
+			Data: map[string]string{util.ChownInitImageConfigmapNameKey: "busybox:123"},
 		}
 
-		client := fake.NewClientBuilder().WithScheme(runtime.NewScheme()).Build()
+		client := fake.NewClientBuilder().WithScheme(getTestScheme()).WithObjects(addImagesConfigmap).Build()
```

Comment:
Why is the "addImagesConfigmap" needed for this test?
I think it can be removed here

Reply:
done.